onnx/defs/schema.h

```diff
@@ -194,7 +198,10 @@ class OpSchema final {
    * @brief Verifies if a NodeProto matches the pattern specified in
    * the schema.
    */
-  void Verify(const NodeProto& node) const;
+  void Verify(
```

Comment:
Can we leave this context independent? And move the verifyNode in the checker itself?

Reply:
The context is used to report errors. I changed the logic to report more than one error at a time, but that requires using something (the context in this case) to accumulate error messages.

Reply:
I've moved the operator schema verification to checker.cc.